TourMate/TourMate/Frontend/Plan/Views/Common/SearchViewModel.swift

```diff
@@ -20,7 +20,7 @@ class SearchViewModel: ObservableObject {
 
     private var cancellableSet: Set<AnyCancellable> = []
 
-    init(locationService: LocationService = RealLocationService()) {
+    init(locationService: LocationService = MockLocationService()) {
```

Comment:
Oh, why are changing to use the mock service now?

Reply:
Oh, i didn't want to make so many api calls when testing

Reply:
Oh ok, maybe should keep it as mock then haha